balance.go

```diff
@@ -12,13 +24,15 @@ type BalanceParams struct {
 // Balance is the resource representing your Stripe balance.
 // For more details see https://stripe.com/docs/api/#balance.
 type Balance struct {
-	Available []*Amount `json:"available"`
-	Livemode  bool      `json:"livemode"`
-	Pending   []*Amount `json:"pending"`
+	Available       []*Amount `json:"available"`
+	ConnectReserved []*Amount `json:"connect_reserved"`
+	Livemode        bool      `json:"livemode"`
+	Pending         []*Amount `json:"pending"`
 }
 
 // Amount is a structure wrapping an amount value and its currency.
 type Amount struct {
-	Value    int64    `json:"amount"`
-	Currency Currency `json:"currency"`
+	Value       int64                       `json:"amount"`
+	Currency    Currency                    `json:"currency"`
+	SourceTypes map[BalanceSourceType]int64 `json:"source_types"`
```

Comment:
I know it was like this already, but we should consider alphabetizing this list just to make it like all the other ones. (That could also easily be pushed as a separate commit to `master` though.)

Reply:
fixed it, I totally did not notice at first